chaos_genius/views/anomaly_data_view.py

```diff
@@ -4,6 +4,7 @@
 import io
 import time
 from datetime import date, datetime, timedelta
+from collections import defaultdict
```

Comment:
Import order was wrong here.

```suggestion
from collections import defaultdict
from datetime import date, datetime, timedelta
```

Reply:
Fixed.